Makefile

```diff
@@ -53,6 +53,7 @@ devssl: build/manifest.json python
 
 .PHONY: devdata
 devdata: python
+	@tox -qe dev --run-command 'python3 -m h.scripts.init_db --create --stamp'
```

Comment:
`tox.ini` used to call `make_db` when running `tox -e dev` but it no longer does so, so the `devdata` command now needs to do that itself.